app/customer.py

```diff
@@ -0,0 +1,42 @@
+from dataclasses import dataclass
+from app.shop import Shop
+from typing import List
+
+
+@dataclass
+class Customer:
+    argus: dict
+    name: str = None
+    product_cart: dict = None
+    location: list = None
+    money: int | float = None
+    car: dict = None
+    cheapest_shop: Shop | None = None
+
+    def init_attrs(self) -> None:
+        for key, value in self.argus.items():
+            setattr(self, key, value)
```

Comment:
Just create regular init. To know what should be in argus I should do a lot of additional work. It's a bad practice